org.pgadmin.pgadmin4.yml

```diff
@@ -0,0 +1,145 @@
+      - --sbindir=${FLATPAK_DEST}/bin
+      - --disable-static
+      - --disable-rpath
+    sources:
+      - type: archive
+        url: https://kerberos.org/dist/krb5/1.21/krb5-1.21.2.tar.gz
+        sha256: 9560941a9d843c0243a71b17a7ac6fe31c7cebb5bce3983db79e52ae7e850491
+        x-checker-data:
+          type: html
+          url: https://kerberos.org/dist/
+          version-pattern: Kerberos V5 Release ([\d\.-]*) - current release
+          url-template: https://kerberos.org/dist/krb5/$version0.$version1/krb5-$version.tar.gz
+
+  - name: python3-bcrypt
+    buildsystem: simple
+    build-commands:
+      - pip3 install --prefix=/app bcrypt-4.1.2-cp39-abi3-manylinux_2_28_x86_64.whl
+    sources:
+      - type: file
+        url: https://files.pythonhosted.org/packages/42/9d/a88027b5a8752f4b1831d957470f48e23cebc112aaf762880f3adbfba9cf/bcrypt-4.1.2-cp39-abi3-manylinux_2_28_x86_64.whl
+        sha256: 2a298db2a8ab20056120b45e86c00a0a5eb50ec4075b6142db35f593b97cb3fb
+        x-checker-data:
+          type: pypi
+          name: bcrypt
+
+  - name: python3-pycparser
+    buildsystem: simple
+    build-commands:
+      - pip3 install --prefix=/app pycparser-2.21-py2.py3-none-any.whl
+    sources:
+      - type: file
+        url: https://files.pythonhosted.org/packages/py2.py3/p/pycparser/pycparser-2.21-py2.py3-none-any.whl
+        sha256: 8ee45429555515e1f6b185e78100aea234072576aa43ab53aefcae078162fca9
+        x-checker-data:
+          type: pypi
+          name: pycparser
+
+  - name: python3-cffi
+    buildsystem: simple
+    build-commands:
+      - pip3 install --prefix=/app cffi-1.16.0-cp311-cp311-manylinux_2_17_x86_64.manylinux2014_x86_64.whl
+    sources:
+      - type: file
+        url: https://files.pythonhosted.org/packages/9b/89/a31c81e36bbb793581d8bba4406a8aac4ba84b2559301c44eef81f4cf5df/cffi-1.16.0-cp311-cp311-manylinux_2_17_x86_64.manylinux2014_x86_64.whl
+        sha256: 7b78010e7b97fef4bee1e896df8a4bbb6712b7f05b7ef630f9d1da00f6444d2e
+        x-checker-data:
+          type: pypi
+          name: cffi
+
+  - name: python3-cryptography
+    buildsystem: simple
+    only-arches:
+      - x86_64
+    build-commands:
+      - pip3 install --prefix=/app cryptography-42.0.5-cp37-abi3-manylinux_2_28_x86_64.whl
+    sources:
+      - type: file
+        url: https://files.pythonhosted.org/packages/64/f7/d3c83c79947cc6807e6acd3b2d9a1cbd312042777bc7eec50c869913df79/cryptography-42.0.5-cp37-abi3-manylinux_2_28_x86_64.whl
+        sha256: f8837fe1d6ac4a8052a9a8ddab256bc006242696f03368a4009be7ee3075cdb7
+        x-checker-data:
+          type: pypi
+          name: cryptography
+
+  - python3-requirements_filtered.yaml
+
+  - name: pgadmin4-server
+    buildsystem: simple
+    build-commands:
+      - ar x pgadmin-server.deb
+      - tar -xf data.tar.zst
+      - rm -rf usr/pgadmin4/venv
+      - cp -r usr/* /app/
+      - sed -i "233 i DATA_DIR = os.environ['XDG_CONFIG_HOME']" /app/pgadmin4/web/config.py
```

Comment:
isn't there a better way than sed?

Reply:
This would be much better I think https://github.com/pgadmin-org/pgadmin4/pull/7359

I'm not sure if the other sed command could be removed, if we place the folders differently @faveoled might know.

Reply:
patches are always better. because patches can fail. while sed will just move on silently.

Reply:
I guess my question was more along the lines of, can we change the folder structure so that we don't need, this at all:

```
- sed -i "43 i pgadminFile = '/app/pgadmin4/web/pgAdmin4.py'" ${FLATPAK_DEST}/pgadmin4/bin/src/js/pgadmin.js
```
As the default is

```
let pgadminFile = '../web/pgAdmin4.py';
```

I'm also now wondering why we don't need to patch 

```
let configFile = '../web/config.py';
```